sdk/search/search-documents/src/indexModels.ts

```diff
@@ -994,3 +1027,26 @@ export interface VectorSearchOptions<TModel extends object> {
    */
   filterMode?: VectorFilterMode;
 }
+/** The threshold used for vector queries. */
+export interface BaseVectorThreshold {
```

Comment:
I don't think this base is needed or does anything, does it? If you remove it (and the two interfaces below no longer extend this) does anything change or break?

Reply:
This is a rename of a generated type. I'll open an issue to update all such types to be emitted as Autorest transforms.